README.md

```diff
@@ -2,3 +2,73 @@ nih-wayfinding
 ==============
 
 NIH Wayfinding Tools (Prototype)
+
+## Local Development
+
+A combination of Vagrant 1.5+ and Ansible 1.6+ is used to setup the development environment for this project. It creates the following virtual machines:
+
+- `app`
+- `services`
+
+The `app` virtual machine contains an instance of the Angular/Scala application and `services` contains the PostgreSQL database and other services.
+
+Use the following command to bring up a local development environment:
+
+```bash
+$ vagrant up
+```
+
+After provisioning is complete, you can login to the application server or services server:
+
+```bash
+$ vagrant ssh [app|services]
+```
+
+**Note**: If you get an error that resembles the following, try logging into the `app` virtual machine again for the group permissions changes to take effect:
```

Comment:
This note is specific to `envdir`. If you guys don't end up using it, it should be safe to remove.